### PR TITLE
Add method to convert canvas to ascii art

### PR DESCRIPTION
**Pull request by nvaytet**
Use
```Py
p = pp.plot(da)
p.canvas.to_ascii()
```
Screenshot at 2023-02-13 16-35-20

### CODE REVIEW

**Comment by nvaytet** [comment]
I don't know if it's best to return the string, or to print inside the function.

File: src/plopp/backends/matplotlib/canvas.py [hunk 1]
```diff
@@ -116,6 +116,37 @@ def to_image(self):
...
+        height = int(width * ratio)
+        im = im.resize((width, height))
+        # Convert to grayscale
+        im = im.convert("L")
+        # Convert to ASCII
+        pixels = im.getdata()
+        div = 256 / len(ASCII_CHARS)
+        text = "".join([ASCII_CHARS[int(pixel / div)] for pixel in pixels])
+        pixel_count = len(text)
+        return "\n".join(text[i:(i + width)] for i in range(0, pixel_count, width))
```
Comment (nvaytet) [inline src/plopp/backends/matplotlib/canvas.py:148]: I don'tknow if it's best to return the string or print inside the function.

**Comment by SimonHeybrock** [comment]
I don't feel the result is very useful? Are you sure you want to add and maintain code for this feature?

**Comment by nvaytet** [comment]
No it's not very useful, would be better if it would make a@the axes with border characters and actual numbers next to the ticks.

**Pull request closed without merging**